### ISSUE TITLE
Node exporter install is broken on ubuntu ami

### ISSUE THREAD

**Issue by AdamNuclear** [scylladb/scylla-monitoring#2230]
Scylla Monitor 4.6.2

Broken node_exporter install script:

```
root@ip-10-107-3-73:/home/support# /home/ubuntu/node_exporter_install
Traceback (most recent call last):
  File "/home/ubuntu/node_exporter_install", line 61, in <module>
    data = curl('https://github.com/prometheus/node_exporter/releases/download/v{version}/node_exporter-{version}.linux-amd64.tar.gz'.format(version=VERSION), byte=True)
NameError: name 'curl' is not defined
Debug log created: /var/tmp/scylla/node_exporter_install-23645-debug.log
root@ip-10-107-3-73:/home/support# 
```

**Comment by mykaul** [comment]
CC @benipeled 

It's unclear to me why node_exporter is not part of the image.

**Comment by amnonh** [comment]
node_exporter installation is part of the image creation and got broken. it's something I need to fix for 4.7

**Comment by mykaul** [comment]
@amnonh - was it fixed in 4.7?

**Comment by amnonh** [comment]
> @amnonh - was it fixed in 4.7?

I've managed to create AWS image, and it was fine there, I couldn't create GCP image so I'm not sure, cc @benipeled

**Comment by amnonh** [comment]
Solved in 4.7.0
